doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md

```diff
@@ -9,6 +9,7 @@ This table describes fields of the Receipt Request applicable to the French mark
 | Field name                | Data type | Default Value Mandatory Field | Description                                                                                              | Version |
 |---------------------------|-----------|-------------------------------|----------------------------------------------------------------------------------------------------------|---------|
 | `cbReceiptAmount`       |`Decimal`?	| null<br>mandatory	                | Total receipt amount incl. taxes (gross receipt amount).                                                 | 1.2      |
+| `ftPosSystemId          |guid/string |null<br>mandatory	                |	This field identifies and documents the type and software version of the PosSystem sending the request. It is used for audits and as a base for commission calculation. The PosSystem itself has to be created in the portal and its ID can be implemented as a constant value by the PosCreator. | 1.2      |
```

Comment:
A second ` is needed to make this visualize highlighted. Additionally  we should highlight the second column

![image](https://user-images.githubusercontent.com/7176615/94651507-d7c3a400-02f8-11eb-8821-db5d7fc8d410.png)

```
| `ftPosSystemId`              | `guid/string`         | null<br />mandatory                         | This field identifies and documents the type and software version of the PosSystem sending the request. It is used for audits and as a base for commission calculation. The PosSystem itself has to be created in the portal and its ID can be implemented as a constant value by the PosCreator.                                                  | 1.2             |
```